spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppDeployer.java

```diff
@@ -153,4 +154,11 @@
 	default String getLog(String id) {
 		throw new UnsupportedOperationException("'getLog' is not implemented.");
 	}
+	/**
+	 * Scale an app to the desired count.
+	 *
+	 * @param id the app deployment id, as returned by {@link #deploy}
+	 * @param desiredCount the desired number of instances.
+	 */
+	void scale(String id, int desiredCount);
```

Comment:
Should we make this a default method same as `getLog`? Then it would not break anything immediately.

Reply:
Yes, leave those tests disabled. We can then require those when time is right.